src/main/resources/search-terms.json

```diff
@@ -0,0 +1,80 @@
+[
```

Comment:
I think we need to remove this version of the file in `/src/main/resources` and keep the one in `/src/test/resources`